packages/fxa-settings/src/pages/Signin/index.test.tsx

```diff
@@ -151,4 +166,47 @@ describe('Signin', () => {
       expect(GleanMetrics.login.forgotPassword).toBeCalledTimes(1);
     });
   });
+
+  describe('form submission', () => {
+    describe('with a password', () => {
+      it('emits the expected metrics events', async () => {
+        renderWithLocalizationProvider(
+          <Signin
+            email={MOCK_EMAIL}
+            isPasswordNeeded={true}
+            serviceName={MOCK_SERVICE}
+          />
+        );
+        submit();
+        await waitFor(() => {
+          expect(GleanMetrics.login.submit).toHaveBeenCalledTimes(1);
+          // the submission handling code hasn't been implemented when the
+          // event was added so a success will follow a a submit.
```

Comment:
not worth rerunning tests for this, but spotted typo `a a`

Reply:
Thanks.  I really need to pipe my comments through a spell and grammar checker.  😅